From f2d14163fb12df8a81623191bd2495b6f09f5d52 Mon Sep 17 00:00:00 2001
From: David Andrey <david.andrey@netmodule.com>
Date: Fri, 4 Jan 2013 14:34:34 +0100
Subject: [PATCH 14/21] ARM: xilinx: BSP: fix return value in pin muxing

Fix the return value warning in the pin muxing
function for NAND and QSPI

Signed-off-by:David Andrey david.andrey@netmodule.com
---
 board/netmodule/zynq4ethernet/board.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/board/netmodule/zynq4ethernet/board.c b/board/netmodule/zynq4ethernet/board.c
index 2d352c8..c35d764 100644
--- a/board/netmodule/zynq4ethernet/board.c
+++ b/board/netmodule/zynq4ethernet/board.c
@@ -118,7 +118,7 @@ int board_late_init (void)
 /**
  * Set pin muxing for NAND access
  */
-static int set_mio_mux_nand(){
+static void set_mio_mux_nand(){
 #define SLCRBASE   0xF8000000
 #define MIOMUXBASE 0xF8000700
 #define NANDMUX0 0x0000001610
@@ -150,7 +150,7 @@ static int set_mio_mux_nand(){
 /**
  * Set the pin muxing for QSPI NOR access
  */
-static int set_mio_mux_qspi(){
+static void set_mio_mux_qspi(){
 #define SLCRBASE   0xF8000000
 #define MIOMUXBASE 0xF8000700
 #define QSPIMUX 0x0000000602
-- 
1.7.4.1

